src/ODEInterface.jl

```diff
@@ -161,11 +161,12 @@ end
   """
 const solverInfo = Vector{SolverInfo}() 
 
-
+# Work around syntax changes in v0.6 without requiring Compat
+eval(Expr(:abstract, :(AbstractArgumentsODESolver{FInt} <: Any)))
 """
   Ancestor for all types storing arguments for ODE-(C-/Fortran-)solvers.
-  """
-abstract AbstractArgumentsODESolver{FInt} <: Any
+"""
+AbstractArgumentODESolver
```

Comment:
No, the doc strings won't work with a macro after them (I think they break with `@compat` also.